lib/OpenQA/WebSockets.pm

```diff
@@ -117,12 +121,6 @@ sub _setup {
 
     OpenQA::Setup::read_config($self);
     setup_log($self);
-
-    Mojo::IOLoop->recurring(
-        380 => sub {
-            log_debug('Resetting worker status table');
-            $self->status->worker_status({});
```

Comment:
Likely the whole `worker_status` property could be removed because all uses of it I found seemed rather useless. But that's outside of this PR's scope.

Reply:
ok. To not forget that I will prepare a followup

Reply:
created https://github.com/os-autoinst/openQA/pull/3589